app/renderer/components/task_suite.js

```diff
@@ -8,7 +8,7 @@ const AddTask = require('./add_task');
 const AppStatus = require('../app_status');
 
 module.exports = {
-    props: ['suite', 'index'],
+    props: ['suites', 'index'],
```

Comment:
a question, why do you need all the suites here? I assume because it is needed for the library. Can you think of a solution where the task drag is out of the task suite component so a suite doesn't need all the other suites?

I'm ok with your solution, I want to know for possible improvements in the code